src/js/features/metadata/constants.js

```diff
@@ -35,5 +35,41 @@ export const DELETE_OFFER_SUCCEEDED = 'METADATA/DELETE_OFFER_SUCCEEDED';
 export const DELETE_OFFER_FAILED = 'METADATA/DELETE_OFFER_FAILED';
 
 // Mapping
-export const PAYMENT_METHODS = ['Cash (In person)', 'Bank Transfer', 'International wire'];
+export const POPULAR_PAYMENT_METHODS = [1, 2, 3];
```

Comment:
Could you rename this `POPULAR_PAYMENT_METHODS_INDEXES` so it's clear that this list is actually just the indexes and not the actually methods